avaframe/com1DFA/com1DFA.py

```diff
@@ -1098,6 +1098,12 @@ def initializeSimulation(cfg, outDir, demOri, inputSimLines, logName):
     entreainableMass = np.nansum(fields["entrMassRaster"] * dem["areaRaster"])
     log.info("Mass available for entrainment: %.2f kg" % (entreainableMass))
 
+    log.debug("Initializing detrainment area")
+    detRaster, reportAreaInfo = initializeDetrainment(
+        cfgGen, dem, simTypeActual, inputSimLines["resLine"], reportAreaInfo, thresholdPointInPoly
+    )
+    fields["detRaster"] = detRaster
+
```

Comment:
Since we want the detrainment to be coupled to the resistance areas, would combining it with the resistance initialization make more sense?

Reply:
So, I added the detrainment-initialization into the resistance-initialization function:
First, resistanceRaster initialized,
if detrainment is switched on: detrainment is initialized

and for only detrainment (without resistance) another flag can set resistanceRaster to zeros (only if detrainment is switched on).